### PR TITLE
Gutenberg in Calypso Page Editor Specs

### PR DESCRIPTION
**Pull request by JavonDavis**
Related: https://github.com/Automattic/wp-e2e-tests/issues/1620

Converting most Page specs to run against Gutenberg in Calypso

### CODE REVIEW

File: specs/wp-gutenberg-post-editor-spec.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ before( async function() {
 describe( `[${ host }] Gutenberg Editor: Posts (${ screenSize })`, function() {
 	this.timeout( mochaTimeOut );
 
-	describe( 'Public Posts: Preview and Publish a Public Post @parallel', function() {
+	describe.only( 'Public Posts: Preview and Publish a Public Post @parallel', function() {
```
Comment (alisterscott) [inline specs/wp-gutenberg-post-editor-spec.js]: This is causing the CircleCI results to only run one spec

File: specs/wp-calypso-gutenberg-page-editor-spec.js [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+
+before( async function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = await driverManager.startBrowser();
+} );
+
+describe( `[${ host }] Editor: Pages (${ screenSize })`, function() {
+	this.timeout( mochaTimeOut );
+
+	describe( 'Public Pages: @parallel @jetpack', function() {
```
Comment (alisterscott) [inline specs/wp-calypso-gutenberg-page-editor-spec.js]: The preview step is not working:

```
 TimeoutError: Timed out waiting for element with css selector of '.type_page' to be present and displayed
Wait timed out after 20276ms
```

**Comment by Stojdza** [comment]
I've pushed two minor changes, so tests are passing now. Merging now to avoid conflicts.

**Comment by JavonDavis** [comment]
Thanks for patching this for me @Stojdza

**Pull request merged**